extensions/ql-vscode/src/view/variant-analysis/VariantAnalysis.tsx

```diff
@@ -63,8 +61,16 @@ export function VariantAnalysis({
   const [selectedRepositoryIds, setSelectedRepositoryIds] = useState<number[]>(
     [],
   );
-  const [filterSortState, setFilterSortState] =
-    useState<RepositoriesFilterSortState>(defaultFilterSortState);
+  useTelemetryOnChange(
+    selectedRepositoryIds,
+    "variant-analysis-selected-repository-ids",
```

Comment:
Can you explain why this usage has no `debounceTimeoutMillis`, but the one below does?

Reply:
Good point. This one probably should have a debounce too, as the user could click a lot of checkboxes in quick succession.